README.md

```diff
@@ -70,18 +70,17 @@ The agent is configured using the environment variables presented in the
 following table. Note that any unset variables will be replaced with their
 default values.
 
-| Variable                    | Description                                                       | Default                          |
-| --------------------------- | ----------------------------------------------------------------- | -------------------------------- |
-| MF_LICENSE_LOG_LEVEL        | Agent log level                                                   | "error"                          |
-| MF_LICENSE_SERVICE_URL      | License service URL                                               | "http://localhost:8180/licenses" |
-| LICENSE_FILE                | File to store license in                                          | "./license"                      |
-| MF_LICENSE_CLIENT_TLS       | HTTP client for License service                                   | "false"                          |
-| MF_LICENSE_AGENT_CERT       | Agent  TLS cert                                                   |                                  |
-| MF_AGENT_SERVER_KEY         | Agen  TLS cert key                                                |                                  |
-| MF_AGENT_PORT               | Agent port                                                        |                                  |
-| MF_AGENT_LOAD_RETRY_SECONDS | Retry period in seconds to wait between two sync calls to service | 60                               |
-| MF_AGENT_LICENSE_ID         | Agent License ID                                                  |                                  |
-
-The Agent fetches the License using it's ID, and exposes HTTP API for validation. You can se the License validation flow on the diagram below:
+| Variable                    | Description                                                       | Default                                  |
+| --------------------------- | ----------------------------------------------------------------- | ---------------------------------------- |
+| MF_LICENSE_LOG_LEVEL        | Agent log level                                                   | "error"                                  |
+| MF_LICENSE_SERVICE_URL      | License service URL                                               | "http://localhost:8180/licenses/devices" |
```

Comment:
Port should be `8111`, right? Maybe this could simply `localhost:8111`?

Reply:
It can not be `localhost:8111` because it is an HTTP endpoint, but it should be `8111` port. Let me fix it.